qiskit/transpiler/passes/layout/dense_layout.py

```diff
@@ -125,14 +119,25 @@ def _best_subset(self, num_qubits, num_meas, num_cx):
         if num_qubits == 0:
             return []
 
+        adjacency_matrix = rustworkx.adjacency_matrix(coupling_map.graph)
+        reverse_index_map = {v: k for k, v in enumerate(coupling_map.graph.nodes())}
+
+        error_mat, use_error = _build_error_matrix(
+            coupling_map.size(),
+            reverse_index_map,
+            backend_prop=self.backend_prop,
+            coupling_map=self.coupling_map,
```

Comment:
Just a sanity check, should this be `coupling_map` or `self.coupling_map`?

Reply:
I think it should be `self.coupling_map`. It's using the global connectivity graph to look up error rates for the full device (either in `target` or `backend_prop`).  We can only look up the error rates from the global graph's indices not the local connected component subgraphs' indices. The `reverse_index_map` value for the `qubit_map` argument is used to map the physical qubit on the full connectivity graph from `self.coupling_map` to the matrix index corresponding to the node index on the connected component subgraph the pass runs on.